test/Depositor.t.sol

```diff
@@ -19,29 +19,10 @@ import "./utils/Stub.sol";
 
```

Comment:
Should we add a test here simulation the a low-value, high-decimal token, just to make sure everything works?

Reply:
[c573013](https://github.com/Cozy-Finance/cozy-safety-module-rewards-manager/pull/55/commits/c57301328cca804d4ea310d12659ee771b7a5803)